README.md

```diff
@@ -614,6 +616,17 @@ Kinvey CLI is a subject to the following caveats:
 
 If problems persist, contact [Kinvey](http://support.kinvey.com).
 
+### Support for Service Environments
+
+Kinvey CLI version 4.1.0 introduces support for environments inside Flex services coinciding with the release of this feature on the backend. When upgrading from a previous Kinvey CLI version, have the following caveats in mind:
+
+- The `flex init` interactive command prompts for service environment selection in version 4.1.0 and later.
```

Comment:
It won't prompt for a svc env if the service contains only one. Should we mention this?

Reply:
Added.